src/Columns/IColumn.cpp

```diff
@@ -9,6 +9,11 @@
 namespace DB
 {
 
+Field IColumn::get(size_t n) const
```

Comment:
I think it's better to move it to header for inline.

Reply:
Then I'll have to include Field.h there. Let's just delete it, `->operator[](0)` is not so bad.